### PR TITLE
releases: use common stylesheet

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Fixes issue #2154

Changes proposed in this pull request:

- Use common stylesheet for release notes pages

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @duck-rh @gregsheremeta @lwrigh

### CODE REVIEW

File: source/_sass/lib/site.sass [hunk 1]
```diff
@@ -373,6 +373,35 @@ div.case-studies
...
+    font-family: 'Open Sans', sans-serif !important
+
+  li
+    font-family: 'Open Sans', sans-serif !important
+
+  a
+    font-family: 'Open Sans', sans-serif !important
+
+  p
+    font-family: 'Open Sans', sans-serif !important
```
Comment (duck-rh) [inline source/_sass/lib/site.sass:402]: You don't need to repeat each tag, you can do instead:
```suggestion
.releases
  h1, h2, h3…
    font-family: 'Open Sans', sans-serif !important
```

**Comment by duck-rh** [comment]
@sandrobonazzola it would be since to finish this one when you have time.

File: source/_sass/lib/site.sass (hunk 1 above)
Comment (sandrobonazzola) [reply source/_sass/lib/site.sass]: done

**Comment by sandrobonazzola** [comment]
Thanks, fixed

**Comment by duck-rh** [comment]
@sandrobonazzola I think you can squeeze the commits since it all goes together, and also rebase since there are conflicts now.

**Pull request merged**